packages/react-cookie/src/manager.ts

```diff
@@ -0,0 +1,65 @@
+import cookie from 'cookie';
+import {hasDocumentCookie} from './utilities';
+import {Cookies, CookieValue} from './types';
+
+export {CookieContext} from './context';
+
+export class CookieManager {
```

Comment:
tbh I'm not super sure why we aren't using `universal-cookies` as our manager, does it clash with our expected APIs in some way? It's simple enough that it's not a huge deal for us to have our own but if there's no real reason not to we might as well use it.